gym-unity/gym_unity/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Version of the library that will be used to upload to pypi
-__version__ = "0.20.dev0"
```

Comment:
Whoops, was that user error or a problem in the script?

Reply:
user error